lib/crawler/keyword/keywords.ex

```diff
@@ -67,6 +74,17 @@ defmodule Crawler.Keyword.Keywords do
     |> Repo.update()
   end
 
+  defp compact_params(params) do
+    params
+    |> Enum.filter(fn {_key, value} -> value !== "" end)
```

Comment:
What if value is nil? Should we do filtering in that case?

Reply:
I'm pretty sure that we can't have a `nil` value since this is input value. That's my understanding so far, correct me if I'm wrong

Reply:
We can't be sure until we check, in the future this method can be called in other place which may produce `nil`, so, if we don't want to see `nil` in our filter query we should filter out such results the same way as we filter out `value !== ""`

Reply:
@Nihisil Fixed in [d990ee8](https://github.com/vnntsu/elixir-ic/pull/44/commits/d990ee8ab9764e85ea2e82e06180e10bc9c8c807)